docker-compose/pattern-1/docker-compose.yml

```diff
@@ -1,20 +1,100 @@
-version: '2'
+version: '2.1'
 services:
-  is_rdbms:
-    build:
-        context: ../
-        dockerfile: pattern-1/database/Dockerfile
+  mysql:
+    image: mysql:5.7.19
+    ports:
+      - 3306
     environment:
-        MYSQL_ROOT_PASSWORD: root
+        MYSQL_ROOT_PASSWORD: wso2carbon
+    volumes:
+      - ./mysql/scripts:/docker-entrypoint-initdb.d
+    healthcheck:
+      test: ["CMD", "mysqladmin" ,"ping", "-h", "localhost"]
+      interval: 10s
+      timeout: 60s
+      retries: 5
   identity-server:
-    build:
-      context: ../
-      dockerfile: pattern-1/identity-server/Dockerfile
+    image: wso2is:5.3.0
+    ports:
+      - 9443
+    healthcheck:
+        test: ["CMD", "curl", "-k", "-f", "https://localhost:9443/carbon/admin/login.jsp"]
+        interval: 10s
+        timeout: 120s
+        retries: 5
     environment:
-      - SLEEP=10
+      - SERVICE_PORTS=9443
+      - VIRTUAL_HOST=https://wso2is
+      - EXTRA_ROUTE_SETTINGS=ssl check verify none
+      - BALANCE=source
+    volumes:
+        - ./identity-server/carbon/repository/components/lib/mysql-connector-java-5.1.34-bin.jar:/home/wso2carbon/wso2is-5.3.0/repository/components/lib/mysql-connector-java-5.1.34-bin.jar
+        - ./identity-server/carbon/repository/conf/carbon.xml:/home/wso2carbon/wso2is-5.3.0/repository/conf/carbon.xml
+        # - ./identity-server/carbon/repository/conf/datasources/master-datasources.xml:/home/wso2carbon/wso2is-5.3.0/repository/conf/datasources/master-datasources.xml
+        - ./identity-server/carbon/repository/conf/tomcat/catalina-server.xml:/home/wso2carbon/wso2is-5.3.0/repository/conf/tomcat/catalina-server.xml
+    depends_on:
+      mysql:
+        condition: service_healthy
+    links:
+        - mysql
+  lb:
+    image: 'dockercloud/haproxy:1.6.3'
     ports:
-      - "9443:9443"
-      - "9763:9763"
-      - "8000:8000"
-      - "10500:10500"
-      
+      - 443:443
+    environment:
+      - TIMEOUT=connect 50000, client 50000, server 50000
+      - STATS_AUTH=stats:stats
+      - |
```

Comment:
What is the use of having a pipe symbol here ?

Reply:
It is used for defining a multi-line string.